src/layout/InfoCard.component.js

```diff
@@ -86,6 +86,8 @@ class InfoCard extends Component {
                 return this.getUserOrgUnitsValue()
             case 'userRoles':
                 return this.getUserRolesValue()
+            case 'emailModal':
```

Comment:
forgot to add the case here. Without this on the view profile page, the email was displaying as "your_email@dhis2.org value not found" 😝